octoprint_mrbeam/migrate.py

```diff
@@ -294,8 +294,11 @@ def _compare_versions(self, lower_vers, higher_vers, equal_ok=True):
         return LooseVersion(lower_vers) < LooseVersion(higher_vers)
 
     def save_current_version(self):
-        self.plugin._settings.set(["version"], self.version_current, force=False)
-        self.plugin._settings.save()
+        if self.plugin._settings.get(["version"]) != self.version_current:
+            self.plugin._settings.set(
+                ["version"], self.version_current, force=True
+            )  # force needed to save it if it wasn't there
```

Comment:
Just a reminder to put the comments on a separate line so we avoid ugly Black formatting.